src/status/Status.ts

```diff
@@ -147,7 +153,10 @@ class Status {
         return await this.readStatus();
       },
       (e, statusInfo) => {
-        if (e != null) return true;
+        if (e != null && !(e instanceof statusErrors.ErrorStatusParse)) {
```

Comment:
Capturing the exception here is going to be a bit a problem. What happens if the file is really corrupted and should be ignored.

This should be rare to happen though?

Reply:
Can we remove this and return this to `if (e != null) return true;`? I don't like catching the `ErrorStatusParse`, this doesn't match the semantics of `waitFor`. How often does this happen?

Reply:
On the laptop when I ran the tests 10 in a row I'd get just under half failing due to parsing errors. I'll revert this for now but we should keep an eye on it.